src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,7 +28,14 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+       
+		this.sequenceNumber=nextSequenceNumber++;
     }
+    
+    
+    public int getSequenceNumber() {
+    	         return sequenceNumber;
+    	     }
```

Comment:
indentation is off here.
check out which keyboard shortcut is useful in formatting the code for you!